src/PixiEditor/ViewModels/Document/Nodes/Effects/PosterizationNodeViewModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using PixiEditor.ChangeableDocument.Changeables.Graph.Nodes.Effects;
+using PixiEditor.ViewModels.Nodes;
+
+namespace PixiEditor.ViewModels.Document.Nodes.Effects;
+
+[NodeViewModel("POSTERIZATION_NODE", "EFFECTS", PixiPerfectIcons.ColorPalette)]
```

Comment:
ColorPalette icon is already used in `Color` node. We are using lucide icons https://lucide.dev/icons/

To add an icon, open icomoon.app, drag [defs.svg](https://github.com/PixiEditor/PixiEditor/blob/master/src/PixiEditor.UI.Common/Fonts/defs.svg) and add your selected icon (you may need to flatten it in some app like Figma)

Give it a name in export, download the font and replace ttf and svg, then run `gen-icons.py`